pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -293,28 +293,90 @@ func (t ResolvedPipelineTask) isCancelled() bool {
 // or Run associated.
 func (t ResolvedPipelineTask) isScheduled() bool {
 	if t.IsCustomTask() {
-		return t.RunObject != nil
+		return t.RunObject != nil || len(t.RunObjects) > 0
```

Comment:
do we need to update the docstring of this function? Same question for other functions which we made changes